src/behaviors/Column.js

```diff
@@ -224,17 +236,17 @@ Column.prototype = {
             return 'unknown';
         }
         var type = this.typeOf(value);
-        var isNumber = ((typeof value) === 'number');
+        //var isNumber = ((typeof value) === 'number');
```

Comment:
Let's remove `getType` and `computeColumnType` entirely. In its place let's provide the functionality as sample code in a demo that looks at the data and sets `schema[*].type`.